### ISSUE TITLE
Inappropriate role and accessible name

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/vaadin-avatar#112]
vaadin-avatar has `role="button"` by default, which indicates to screen readers that it is clickable, even though it is not interactive by default, nor does it have features to make it interactive (other than the possibility to attach clicklisteners that it shares with any other element).

A more appropriate default role would be `role="img"`.

If collaboration engine uses the component in a way that makes it clickable, the role should probably be overriden back to `"button"`.

Also the title attribute is not a particularly reliable way to provide an accessible name for the component. `aria-label` should be used to convey the full name to screen readers.
